client/src/components/ProTx/components/charts/MainChart.js

```diff
@@ -196,98 +179,69 @@ function MainChart({
   /***********************/
 
   if (chartType === 'maltreatment') {
-    // Dummy state data to make the empty plot render.
-    let protxMaltreatmentDistribution = {};
-
-    const prepMaltreatmentPlotData = (
-      selectedGeographicFeaturePrep,
-      maltreatmentTypesPrep,
-      dataPrep,
-      geographyPrep,
-      yearPrep,
-      showRatePrep
-    ) => {
-      const geoid = selectedGeographicFeaturePrep;
-      const maltreatmentTypesList = getMaltreatmentTypeNames(
-        maltreatmentTypesPrep,
-        dataPrep
-      );
-
-      const maltreatmentTypesDataValues = getMaltreatmentSelectedValues(
-        dataPrep,
-        geographyPrep,
-        yearPrep,
-        showRatePrep,
-        geoid,
-        maltreatmentTypesPrep
-      );
+    /**
+     * TODO: Use geoid value instead of selectedArea string value on backend.
```

Comment:
should this be in a ticket (and then ticket written here or in saga)?  or do you want to do this before merging?